README.md

```diff
@@ -34,70 +26,66 @@
 > to set-up Nextflow. Make sure to [test your setup](https://nf-co.re/docs/usage/introduction#how-to-run-a-pipeline)
 > with `-profile test` before running the workflow on actual data.
 
-<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
-     Explain what rows and columns represent. For instance (please edit as appropriate):
-
 First, prepare a samplesheet with your input data that looks as follows:
 
 `samplesheet.csv`:
 
 ```csv
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
+sample,cram,crai
+sample1,test.cram,test.cram.crai
 ```
 
-Each row represents a fastq file (single-end) or a pair of fastq files (paired end).
-
--->
+Each row represents one sample.
 
 Now, you can run the pipeline using:
 
 <!-- TODO nf-core: update the following command to include all required parameters for a minimal example -->
 
 ```bash
-nextflow run nf-core/createpanelofnormals \
+nextflow run qbic-pipelines/createpanelofnormals \
    -profile <docker/singularity/.../institute> \
    --input samplesheet.csv \
    --outdir <OUTDIR>
 ```
 
+To get an overview over all avaialbel parameters, run:
+
+```bash
+nextflow run qbic-pipelines/createpanelofnormals --help
+```
+
 > **Warning:**
 > Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those
 > provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_;
 > see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
 
-For more details and further functionality, please refer to the [usage documentation](https://nf-co.re/createpanelofnormals/usage) and the [parameter documentation](https://nf-co.re/createpanelofnormals/parameters).
+For more details and further functionality, please refer to the [usage documentation](./docs/usage.md) and the [parameter documentation](./nextflow_schema.json). You can list all parameters by running `nextflow run qbic-pipelines/createpanelofnormals --help`.
```

Comment:
Otherwise it's double. See line 54
```suggestion
For more details and further functionality, please refer to the [usage documentation](./docs/usage.md) and the [parameter documentation](./nextflow_schema.json).
```